test/benchmark/suite/render.go

```diff
@@ -260,13 +260,14 @@ func writeSection(writer io.Writer, title string, level int, content string) {
 
 // writeCollapsibleSection writes one collapsible section in Markdown style.
 func writeCollapsibleSection(writer io.Writer, title string, content []byte) {
-	_, _ = fmt.Fprintln(writer, fmt.Sprintf(`
+	summary := fmt.Sprintf("```plaintext\n%s\n```", content)
+	_, _ = fmt.Fprintf(writer, `
 <details>
 <summary>%s</summary>
 
 %s
 
-</details>`, title, fmt.Sprintf("```plaintext\n%s\n```", content)))
+</details>\n`, title, summary)
```

Comment:
```suggestion
</details>
`, title, summary)
```

the `"\n"` will be rendered as literal \n